sql-statements/sql-statement-alter-user.md

```diff
@@ -91,6 +93,43 @@ ALTER USER 'newuser' ACCOUNT LOCK;
 Query OK, 0 rows affected (0.02 sec)
 ```
 
+修改 `newuser` 的属性：
+
+```sql
+ALTER USER 'newuser' ATTRIBUTE '{"newAttr": "value", "deprecatedAttr": null}';
+SELECT * FROM user_attributes;
+```
+
+```sql
++-----------+------+---------------------------------------------------+
+| USER      | HOST | ATTRIBUTE                                         |
++-----------+------+---------------------------------------------------+
+| newuser   | %    | {"newAttr": "value", "deprecatedAttr": null}      |
++-----------+------+---------------------------------------------------+
+1 rows in set (0.00 sec)
+```
+
+
+通过 `ALTER USER ... COMMENT` 修改用户 `newuser` 的注释：
+
+```sql
+ALTER USER 'newuser' COMMENT 'Here is the comment';
+```
+
+```
+Query OK, 0 rows affected (0.02 sec)
+```
```

Comment:
````suggestion
```sql
ALTER USER 'newuser' COMMENT 'Here is the comment';
SELECT * FROM user_attributes;
```

```sql
+-----------+------+---------------------------------------------------+
| USER      | HOST | ATTRIBUTE                                         |
+-----------+------+---------------------------------------------------+
| newuser   | %    | {"comment": "Here is the comment", "newAttr": "value", "deprecatedAttr": null}      |
+-----------+------+---------------------------------------------------+
1 rows in set (0.00 sec)
```
````